### PR TITLE
Servant 0.18 & increase upper-bounds

### PR DESCRIPTION
**Pull request by ocharles**

### CODE REVIEW

**Comment by ocharles** [comment]
@chshersh @vrom911 ping. I'm happy to help maintain this library if you want!

**Comment by chshersh** [comment]
Hi @ocharles! Thanks for reaching out 🤗 
We don't maintain this library anymore because we don't work at Holmusk.
I can recommend reaching out to @arbus or writing an email to `tech@holmusk.com` if you're interested in maintaining this library.

**Comment by ocharles** [comment]
Thanks @chshersh, I'll do just that!

File: servant-hmac-auth.cabal
Comment (turboMaCk) [inline servant-hmac-auth.cabal:103]: is there a problem with supporting README.lhs? I think it would be beneficial to keep this executable.

Comment (ocharles) [reply servant-hmac-auth.cabal:103]: Sorry, I imagine it didn't build. I can get that building again.

Comment (turboMaCk) [reply servant-hmac-auth.cabal:103]: it's not the highest priority. I was mainly just curious. We can even make it work after 0.18 support is out. This will be up to @kahlil29 to make call on.

**Comment by kahlil29** [comment]
Update: It seems like Travis CI has shifted domains since we last ran a CI job for this project! 
I'm trying to get CI back up and running before I can merge this.

**Comment by kahlil29** [comment]
Update : I plan on shifting CI from Travis to GitHub Actions and have it done by approx this week.

**Review by turboMaCk — APPROVED** [review]

**Comment by ocharles** [comment]
Superseded by #49

**Pull request closed without merging**
